CHANGELOG.md

```diff
@@ -213,185 +214,4 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Add the --depth parameter for the -R and --tree options from [jorpic](https://github.com/jorpic)
 - Add the directory-order flag
-- Add a basic unicode support from [loewenheim](https://github.com/loewenheim)
```

Comment:
Thanks for the PR, but looks like a lot of the CHANGELOG got deleted somehow. Could you undo that and we should be good to go. :D